tests/datasets-memcap-01/README.md

```diff
@@ -0,0 +1,14 @@
+Test Description
+================
+
+This test demonstrates that the memcap settings take the string length into account in master.
```

Comment:
`master` branch will change, please specify the version 

Also test.yaml has `lt-version: 8` so it is not master, is it ?

Reply:
> `master` branch will change, please specify the version
> 

ok

> Also test.yaml has `lt-version: 8` so it is not master, is it ?

I exchanged the READMEs of the two tests 🤦🏽‍♀️